Anytype/Sources/Models/Documents/Document/BaseDocument+Extensions.swift

```diff
@@ -76,4 +76,26 @@ extension BaseDocumentProtocol {
             .removeDuplicates()
             .eraseToAnyPublisher()
     }
+    
+    func buildParsedRelationsForType() -> ParsedRelations {
```

Comment:
If the client reads this field very frequently, it will cause performance issues. The other parsedRelations are updated via the update diff. I think this field should be included in the design of the BaseDocument update.

Reply:
You are also using `Container.shared`, which means the document is designed incorrectly for this field.